dm/ctl/ctl.go

```diff
@@ -116,6 +116,7 @@ func PrintHelp(args []string) {
 
 // Start starts running a command
 func Start(args []string) {
+	commandMasterFlags.workers = commandMasterFlags.workers[:0]
```

Comment:
add a method like `Reset` for `CommandMasterFlags` to do this? (we may add more fields into `CommandMasterFlags` later)